cmd/pg/backup_push.go

```diff
@@ -54,6 +55,10 @@ var (
 			if useRatingComposer {
 				tarBallComposerType = postgres.RatingComposer
 			}
+			useSimpleComposer = useSimpleComposer || viper.GetBool(internal.UseSimpleComposerSetting)
```

Comment:
Well, now this is not actually a composer type, but a flag to enable the low memory usage. So I suggest to rename  this to `--reduce-memory-usage` or something like this since it is not a new composer type.